azurerm/resource_arm_netapp_snapshot_test.go

```diff
@@ -0,0 +1,340 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMNetAppSnapshot_basic(t *testing.T) {
+	resourceName := "azurerm_netapp_snapshot.test"
+	ri := tf.AccRandTimeInt()
+	location := testLocation()
```

Comment:
This will need to be updated to match the existing tests:
```
data := acceptance.BuildTestData(t, "azurerm_netapp_snapshot", "test")
```

Reply:
fixed